src/styles/blocks/html/body.scss

```diff
@@ -0,0 +1,10 @@
+.html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
The class name `.body` is not recommended for the same reason as `.html`. It's better to use a class name that reflects the block or element you are styling or apply these styles directly to the `body` tag if they are meant to be global.